### PR TITLE
xq: update to 1.2.5, add INSTALL.msg

### PR DESCRIPTION
**Pull request by realcharmer**
[ci skip]
No template change.

### CODE REVIEW

**Comment by classabbyamp** [comment]
hm i'm not sure that's necessary

**Comment by realcharmer** [comment]
I disagree. People and existing scripts will expect the binary to be `xq` by default, not `xq-go`. This might and likely will break scripts, workflows and introduce unnecessary debugging by users. This has already happened to my friend who used the package and I had to explain what's up. You'd normally parse a file, pipe it to xq and nothing would happen, because it would run the default `/usr/sbin/xq`, which gives absolutely no output in such case, not even when running `xq --help`.
This message, if anything, might prevent such scenarios.

File: srcpkgs/xq/INSTALL.msg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Due to conflicting binary names, the `xq` binary is renamed to `qx-go`
```
Comment (chrysos349) [inline srcpkgs/xq/INSTALL.msg:1]: `qx-go` or `xq-go` ?

Comment (realcharmer) [reply srcpkgs/xq/INSTALL.msg]: Oops, thanks for catching this. Fixed.

**Comment by classabbyamp** [comment]
the other notable example of something like this is `yq-go`. maybe instead the package should be renamed to `xq-go`

**Comment by realcharmer** [comment]
Updated to `1.2.5`.

**Pull request closed without merging**